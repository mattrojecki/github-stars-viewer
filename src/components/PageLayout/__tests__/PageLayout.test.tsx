import { render } from '@testing-library/react'
import { PageLayout } from '..'

describe('PageLayout', () => {
  it('renders component unchanged without children', () => {
    const { container } = render(<PageLayout />)
    expect(container).toMatchSnapshot()
  })

  it('renders component unchanged with children', () => {
    const { container } = render(
      <PageLayout>
        <div>Test</div>
      </PageLayout>,
    )
    expect(container).toMatchSnapshot()
  })
})
