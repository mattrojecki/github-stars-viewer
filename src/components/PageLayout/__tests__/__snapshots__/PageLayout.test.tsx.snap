// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`PageLayout renders component unchanged with children 1`] = `
<div>
  <section
    class="ant-layout css-dev-only-do-not-override-txh9fw"
  >
    <header
      class="ant-layout-header css-dev-only-do-not-override-txh9fw"
    >
      <h1
        class="css-ote8d8"
      >
        ⭐ Github Stars Viewer ⭐
      </h1>
    </header>
    <main
      class="ant-layout-content css-dev-only-do-not-override-txh9fw"
    >
      <div
        class="css-d3x6o2"
      >
        <div>
          Test
        </div>
      </div>
    </main>
    <footer
      class="ant-layout-footer css-dev-only-do-not-override-txh9fw"
    >
      <div
        class="css-1q2lbmy"
      >
        <span
          class="ant-typography css-dev-only-do-not-override-txh9fw"
        >
          Mateusz Rojecki © 2023
        </span>
      </div>
    </footer>
  </section>
</div>
`;

exports[`PageLayout renders component unchanged without children 1`] = `
<div>
  <section
    class="ant-layout css-dev-only-do-not-override-txh9fw"
  >
    <header
      class="ant-layout-header css-dev-only-do-not-override-txh9fw"
    >
      <h1
        class="css-ote8d8"
      >
        ⭐ Github Stars Viewer ⭐
      </h1>
    </header>
    <main
      class="ant-layout-content css-dev-only-do-not-override-txh9fw"
    >
      <div
        class="css-d3x6o2"
      />
    </main>
    <footer
      class="ant-layout-footer css-dev-only-do-not-override-txh9fw"
    >
      <div
        class="css-1q2lbmy"
      >
        <span
          class="ant-typography css-dev-only-do-not-override-txh9fw"
        >
          Mateusz Rojecki © 2023
        </span>
      </div>
    </footer>
  </section>
</div>
`;
